pandas/tests/series/test_indexing.py

```diff
@@ -1838,3 +1838,26 @@ def test_multilevel_preserve_name(self):
         result2 = s.ix['foo']
         self.assertEqual(result.name, s.name)
         self.assertEqual(result2.name, s.name)
+
+    def test_align_broadcast_axis_series(self):
+        # GH 13194
+        # Series.align(DataFrame) tests, 'outer' join
+        df = DataFrame(np.array([[1., 2.], [3., 4.]]), columns=list('AB'))
+        ts = Series([5., 6., 7.])
+        result = ts.align(df, join='outer', axis=0, broadcast_axis=1)
+        expected1 = DataFrame(np.array([[5., 5.], [6., 6.], [7., 7.]]),
+                              columns=list('AB'))
+        expected2 = DataFrame(np.array([[1., 2.], [3., 4.],
+                                        [pd.np.nan, pd.np.nan]]),
+                              columns=list('AB'))
+        self.assertTrue(result[0].equals(expected1))
+        self.assertTrue(result[1].equals(expected2))
+
+        # Series.align(DataFrame) tests, 'inner' join
+        result = ts.align(df, join='inner', axis=0, broadcast_axis=1)
```

Comment:
we can make all tests more comprehensive as "inner" and "outer" outputs the same, and "left" and "right" outputs inversed results.

```
result = df.align(series, ...)
self.assert_frame_equal(result[0], expected1)
self.assert_frame_equal(result[1], expected2)
result = series.align(df, ...)
self.assert_frame_equal(result[0], expected2)
self.assert_frame_equal(result[1], expected1)
```